.github/workflows/deploy-apis-to-staging.yml

```diff
@@ -1566,4 +1573,23 @@ jobs:
    
       - name: Deploy to K8S
         run: |
-          kubectl apply -f k8s/channel-data/stage-airqo-update-channel-data-job.yaml
\ No newline at end of file
+          kubectl apply -f k8s/channel-data/stage-airqo-update-channel-data-job.yaml
+
+  ### apply nginx ingress ###
+  nginx-ingress:
+    name: apply-nginx-ingress
+    needs: [check]
+    if: needs.check.outputs.run_nginx_ingress == 'true'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Login to K8S
+        uses: azure/k8s-set-context@v1
+        with:
+          method: kubeconfig
+          kubeconfig: ${{ secrets.K8S_CONFIG_STAGE }}
+
+      - name: Apply ingress changes
+        run: kubectl apply -f k8s/nginx-ingress/stage-platform-ingress-resource.yaml
```

Comment:
[add apply staging nginx ingress to workflow](https://github.com/airqo-platform/AirQo-api/pull/1073/commits/5b460a36a5d7e44425a2c79840008e6294fbcf3c)